### PR TITLE
Update git-ignore file

### PR DESCRIPTION
**Pull request by ayeshLK**
## Checklist
- [ ] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -23,22 +23,17 @@
...
-*.iml
-*.ipr
-*.iws
+# Java related ignores
+build/
+target/
+.gradle/
 
-# MacOS
-*.DS_Store
```
Comment (chamil321) [inline .gitignore:40]: Why this was removed?

File: .gitignore [hunk 2]
```diff
@@ -23,22 +23,17 @@
...
+# IDE-related files
```
Comment (ayeshLK) [inline .gitignore:31]:
```suggestion
# MacOS
*.DS_Store

# IDE-related files
```

File: .gitignore (hunk 1 above)
Comment (ayeshLK) [reply .gitignore:40]: Updated

File: .gitignore [hunk 3]
```diff
@@ -23,22 +23,20 @@
...
+.gradle/
 
 # MacOS
 *.DS_Store
 
-# Ballerina
-velocity.log*
-*Ballerina.lock
+# IDE-related files
+.idea
```
Comment (ayeshLK) [inline .gitignore:35]:
```suggestion
.idea
*.iml
*.ipr
*.iws
```

File: .gitignore [hunk 4]
```diff
@@ -23,22 +23,20 @@
...
+.code
+.project
+.settings
+.vscode
+
+# Ballerina related ignores
+Ballerina.lock
```
Comment (ayeshLK) [inline .gitignore:42]:
```suggestion
Ballerina.lock
velocity.log*
```

**Review by chamil321 — APPROVED** [review]

**Pull request merged**